config/jobs/kubernetes/sig-node/dra-presubmit.yaml

```diff
@@ -0,0 +1,254 @@
+# GENERATED FILE - DO NOT EDIT!
+#
+# Instead, modify dra.jinja and run `make generate-jobs`.
+presubmits:
+  kubernetes/kubernetes:
+  - name: presubmit-kind-dra
```

Comment:
https://github.com/kubernetes/test-infra/pull/34010#discussion_r1906924332

Now the *file name* follows conventions, but the *job name* doesn't: it should still be called "pull-kind-dra". Yes, the difference is grating.

Ideally both should follow conventions, but if we have to choose for the sake of simplicity, the job name is more important because it is user facing.

Reply:
And the convention for canary job names is `<base name>-canary`, e.g. `pull-kubernetes-kind-dra` and not `canary-kind-dra`.

Reply:
Perhaps we have to hard-code these conventions in the Python script?

Reply:
But then the "kubernetes" part won't be right for other repos :cry:

Reply:
Perhaps we need this in `dra.generate.conf`:
```
files = canary:pull-kubernetes-*-canary,presubmit:pull-kubernetes-*,ci:ci-*
```
and then replace `*` with the job name?

Just a thought, it doesn't have to be like that.

Reply:
done, PTAL

BTW, what do you think about renaming dra.jinja -> dra.generate.jinja to be consistent with the config name?

Reply:
It's a bit long. We need the `.generate.conf` suffix to be unique enough for a global search, but then the suffix on the template IMHO can remain shorter.